extensions/amp-render/1.0/amp-render.js

```diff
@@ -150,10 +177,21 @@ export class AmpRender extends BaseElement {
     });
 
     return dict({
-      'getJson': getJsonFn(this.element),
+      'getJson': this.getFetchJsonFn(),
     });
   }
 
+  /** @override */
+  mutationObserverCallback(mutations) {
+    const isSrcMutated = mutations.some(
+      (mutation) => mutation.attributeName === 'src'
+    );
+    const src = this.element.getAttribute('src');
+    if (isSrcMutated && !isAmpStateSrc(src) && !isAmpScriptUri(src)) {
```

Comment:
Isn't `isAmpStateSrc` and `isAmpScriptUri` handled in `getFetchJsonFn`? Why is it being used to guard the call to `mutateProps`?

Reply:
Thought it would be a minor perf improvement since `mutationObserverCallback` is called on every mutation. On second thoughts, doesn't help much; removed it.